From HEAD Mon Sep 17 00:00:00 2001
From: Hypothesis 6.151.9 <no-reply@hypothesis.works>
Date: Tue, 02 Jun 2026 14:56:28
Subject: [PATCH] Hypothesis: add explicit examples

---
--- ./test_human_html_escape.py
+++ ./test_human_html_escape.py
@@ -36,6 +36,9 @@
 
 # Invariant: Once processed and we split by ;,within each ; there should be a keyword
 @given(generate_html_chars())
+@example(
+    html_text="0",  # or any other generated value
+).via("discovered failure")
 def test_keyword(html_text):
     # Assuming parsed parameter = False
     mapping = defaultdict(int)
